docs/src/dataDisplay/divider/divider.jsx

```diff
@@ -123,6 +123,30 @@ export default class RelaxedSample extends React.Component {
 }`;
 
 export default class ElementsDivider extends React.Component {
+    onBlur() {
+        console.log('Are you sure you are done?');
```

Comment:
Should we add ESLINT suppress for `console.log()` and `window.alert()` calls?

Reply:
Fixed.

Reply:
Looks good.  Resolving.